Documentation/gettingstarted/k8s-install-kops.rst

```diff
@@ -141,30 +132,33 @@ You may be prompted to create a ssh public-private key pair.
 Deleting a Cluster
 ==================
 
-To undo the dependencies and other deployment features in AWS from the ``kops``
-cluster creation, use ``kops`` to destroy a cluster *immediately* with the
+To undo the dependencies and other deployment features in AWS from the kops
+cluster creation, use kops to destroy a cluster *immediately* with the
 parameter ``--yes``:
 
 .. code:: bash
 
         kops delete cluster ${NAME} --yes
 
+Further reading on using Cilium with Kops
+=========================================
+* See the `kops networking documentation <https://kops.sigs.k8s.io/networking/cilium/>`_ for more information on the 
+  configuration options kops offers.
+* See the `kops cluster spec documentation <https://pkg.go.dev/k8s.io/kops/pkg/apis/kops?tab=doc#CiliumNetworkingSpec>`_ for a comprehensive list of all the options
+
+
+
+
```

Comment:
nit: Most of this file seems to use 2 empty lines to separate sections, probably don't need 4?

Reply:
Fixed the spacing to consistent. Thanks for pointing that out.